docs/src/pages/components/progress/CustomizedProgressBars.js

```diff
@@ -1,49 +1,38 @@
 import * as React from 'react';
-import { makeStyles, withStyles } from '@material-ui/core/styles';
-import CircularProgress from '@material-ui/core/CircularProgress';
-import LinearProgress from '@material-ui/core/LinearProgress';
+import { experimentalStyled as styled } from '@material-ui/core/styles';
+import Box from '@material-ui/core/Box';
+import CircularProgress, {
+  circularProgressClasses,
+} from '@material-ui/core/CircularProgress';
+import LinearProgress, {
+  linearProgressClasses,
+} from '@material-ui/core/LinearProgress';
 
-const BorderLinearProgress = withStyles((theme) => ({
-  root: {
+const BorderLinearProgress = styled(LinearProgress)(
+  ({ theme, styleProps = {} }) => ({
```

Comment:
I feel confused, I did not know that we could do the same that we already do in the public components.

But I have some questions.

1. Why `styleProps = {}`, can an empty object ever arrive?

2. Where does `styleProps` appear from? Why in the public components we should create the object.